virt/virtual_machines/virt-manage-vmis.adoc

```diff
@@ -0,0 +1,14 @@
+[id="virt-manage-vmis.adoc"]
+= Managing virtual machine instances
+include::modules/virt-document-attributes.adoc[]
+:context: virt-manage-vmis
+toc::[]
+
+If you have standalone virtual machine instances (VMIs) that were created independently outside of the {VirtProductName} environment, you can manage them using the web console or the CLI.
```

Comment:
s/independently outside of/outside of

s/using/by using

This is also the only place where you do not say "command-line interface (CLI)" but it's the first thing you read in the assembly.